camstack/cams/cred1.py

```diff
@@ -571,6 +577,30 @@ def _fill_keywords(self) -> None:
         self._set_formatted_keyword('GAIN', 1.98)
 
 
+class Ristretto(CRED1):
```

Comment:
Implementing 
```python
def _constructor_finalize(self) -> None:
    etc...
```
in your CRED1 subclass is now mandatory!
Check out the difference between Apapane and Iiwi.